frontend/src/containers/DocumentList.js

```diff
@@ -619,6 +631,7 @@ class DocumentListContainer extends Component {
         currentPage = index;
     }
 
+    this.lastPage = currentPage;
```

Comment:
> lastPage

what's the meaning of this variable?

If it's the last page viewed by user, i would suggest o rename it to `lastViewedPage`.
I am asking this because when i read `lastPage` i think it's about the index of the last page.